VaticanBAV/borget/BAVborget9.xml

```diff
@@ -226,7 +226,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <seg type="rubrication">Titles, nomina sacra, and the first two letters of each stanza</seg>
```

Comment:
just noticed - we do not use "nomina sacra" - which has a different meaning (abbreviated names) - we write "holy names" instead